### DISCUSSION TITLE
share http config/transport with SDK

### DISCUSSION THREAD

**Discussion by ryantxu** [grafana/grafana-csv-datasource#175] [Ideas]
This has some interesting HTTP client config based on standard settings -- it would be great if the standard setting were easy to use from the SDK.

perhaps something like:
https://github.com/grafana/grafana-plugin-sdk-go/blob/master/experimental/rest_client.go#L33

**Comment by marcusolsson** [comment]
Yeah, I looked at https://github.com/alexanderzobnin/grafana-zabbix/blob/master/pkg/httpclient/httpclient.go for inspiration. Didn't know about the experimental REST client, but it's definitely something we should offer in the SDK.
